CMSgov/design-system
# [WNMGDS-2674] Add VRTs for remaining example

## Summary

This is a follow-up to https://github.com/CMSgov/design-system/pull/2959 that rounds out the list of examples.

Note: I've had to disable the `SingleInputDateField` and `Tooltip` examples because they both have dependencies that do not support ES Modules. These will need to be addressed by future tech-debt tickets. It will become a blocker for any team using newer build systems that may rely on ES Modules support (like a team migrating to Astro).

## How to test

`yarn test:browser:examples:update` to build and test, and make sure it didn't write any changes

## Checklist
